.github/workflows/deploy_gcp_litellm_proxy.yaml

```diff
@@ -63,12 +68,17 @@ jobs:
             docker rm litellm_proxy
             docker run -d \
               -v ~/litellm_proxy_config.yaml:/app/config.yaml \
+              -v ~/.gcp_credentials.json:/app/credentials.json \
               -e OPENAI_API_KEY="${{ steps.secrets.outputs.openai-api-key }}" \
+              -e VERTEXAI_PROJECT=${{ secrets.GCP_PROJECT_ID }} \
+              -e VERTEXAI_LOCATION=${{ vars.VERTEX_AI_LOCATION }} \
+              -e VERTEXAI_ENDPOINT=https://${{ vars.VERTEX_AI_LOCATION }}-aiplatform.googleapis.com/v1 \
+              -e GOOGLE_APPLICATION_CREDENTIALS=/app/credentials.json \
               --log-driver=gcplogs \
               --restart always \
               --network aaq-network \
               --name litellm_proxy \
-              ghcr.io/berriai/litellm:main-v1.34.6 --config /app/config.yaml
+              ghcr.io/berriai/litellm:main-v1.40.10 --config /app/config.yaml
```

Comment:
Had to upgrade to use the new `response_format` field.

Reply:
ok! Can we also upgrade the version we use in the Makefile? To keep it all consistent?

Reply:
Ah thanks for catching!